adapter/internal/oasparser/model/open_api.go

```diff
@@ -92,7 +92,7 @@ func (swagger *MgwSwagger) SetInfoOpenAPI(swagger3 openapi3.Swagger) error {
 				productionUrls = append(productionUrls, *endpoint)
 				swagger.xWso2Basepath = endpoint.Basepath
 			} else {
-				logger.LoggerOasparser.Errorf("error encountered when parsing the endpoint under openAPI servers object")
+				logger.LoggerOasparser.Info("Parsing the endpoint under openAPI servers object has failed. ", err)
```

Comment:
info?

Reply:
yes, this is not exactly an error as the endpoint provided in servers is not considered when endpoints are provided in api.yaml or vendor extensions. i.e. Even if the endpoints are provided correctly, this error will be in the logs because we always try to parse the server object, giving a misleading message. To avoid it, I made it an info log, so we can see whether server object was read without any parsing issues.